### DISCUSSION TITLE
leetcode/141

### DISCUSSION THREAD

**Discussion by giscus** [fuxuemingzhu/Leetcode-Solution-All#16] [General]
# leetcode/141

负雪明烛的刷题博客，LeetCode算法题解、刷题模板

https://fuxuemingzhu.cn/leetcode/141.html

**Comment by dfsj66011** [comment]
1. hashtable 不满足 O(1)的空间复杂度
2. 题目还需要找到环的入口，而不仅仅是判定是否有环
